src/features/donation/models/effects/list-donation-multicall.ts

```
@@ -0,0 +1,71 @@
+    .then((finalExecutionOutcomes = undefined) => {
+      console.log("finalExecutionOutcomes", finalExecutionOutcomes);
+
+      const receipts: DirectDonation[] =
+        finalExecutionOutcomes?.reduce(
+          (acc, { status }) => {
+            const decodedReceipt = atob(
+              (status as InformativeSuccessfulExecutionOutcome["status"]).SuccessValue,
+            );
+
+            try {
+              return [...acc, JSON.parse(decodedReceipt) as DirectDonation];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)